t/plugin/limit-conn2.t

```diff
@@ -228,3 +228,91 @@ GET /hello1
 qr/request latency is nil/
 --- grep_error_log_out
 request latency is nil
+
+
+
+=== TEST 7: invalid route: wrong rejected_msg type
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "plugins": {
+                            "limit-conn": {
+                                    "conn": 1,
+                                    "burst": 1,
+                                    "default_conn_delay": 0.1,
+                                    "rejected_code": 503,
+                                    "key": "remote_addr",
+                                    "rejected_msg": true
+                                }
+                        },
+                        "upstream": {
+                            "nodes": {
+                                "127.0.0.1:1980": 1
+                            },
+                            "type": "roundrobin"
+                        },
+                        "uri": "/limit_conn"
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.print(body)
+        }
+    }
+--- request
+GET /t
```

Comment:
We don't need to set request and error_log. They are set in https://github.com/apache/apisix/blob/05fc2306b5f005799e908273cfe11bd5f830459f/t/plugin/limit-conn2.t#L38

Reply:
okay, thanks, done~